packages/fast-usdc/src/exos/settler.js

```diff
@@ -174,8 +175,8 @@ export const prepareSettler = (
         intermediateRecipient: config.intermediateRecipient,
         /** @type {HostInterface<TargetRegistration>|undefined} */
         registration: undefined,
-        /** @type {SetStore<ReturnType<typeof makeMintedEarlyKey>>} */
-        mintedEarly: zone.detached().setStore('mintedEarly'),
+        /** @type {MapStore<ReturnType<typeof makeMintedEarlyKey>, bigint>} */
```

Comment:
What we really want is a MultiSet. Endo Patterns has CopyBag for that. I'm looking into whether we have a durable option.

Reply:
Cool didn't know that was option. Will look out for your reply on what to use here.